docs/changelog.rst

```diff
@@ -6,6 +6,11 @@ Changelog
 `Unreleased`_
 -------------
 
+Fixed
+~~~~~
+
+- Set default User-Agent header in `Case.call`. `#717`_
```

Comment:
Could you please add ".. _#717: https://github.com/schemathesis/schemathesis/issues/717" string to the end of this file where other links are placed? It will make the rendered changelog point to the proper issue location.

Here is a suggestion - with the `rst` syntax, we need double quotes to make this block highlighted (with single quotes, it will be italic)

```suggestion
- Set default User-Agent header in ``Case.call``. `#717`_
```